test/betting/helpers/log.mligo

```diff
@@ -0,0 +1,18 @@
+(* Return str repeated n times *)
+let repeat (str, n : string * nat) : string =
+    let rec loop (n, acc: nat * string) : string = 
+        if n = 0n then acc else loop (abs(n - 1n), acc ^ str)
+    in loop(n, "")
```

Comment:
Move this file into a common directory as it is repeated for both `oracle` & `betting`

Reply:
Done